src/hooks/useSettings.tsx

```diff
@@ -1,56 +1,64 @@
-import { useAsyncStorage } from '@react-native-async-storage/async-storage'
-import { useCallback, useEffect, useState } from 'react'
-import { SHAPESHIFT_URI } from 'react-native-dotenv'
-import { singletonHook } from 'react-singleton-hook'
```

Comment:
ditto only used in a single place, seems sane to me

Reply:
See https://github.com/shapeshift/mobile-app/pull/109#discussion_r2078333976